api/graphql/reservations/reservation_serializers/mixins.py

```diff
@@ -305,10 +306,23 @@ def check_reservation_duration(self, reservation_unit: ReservationUnit, begin, e
                 ValidationErrorCodes.RESERVATION_TIME_DOES_NOT_MATCH_ALLOWED_INTERVAL,
             )
 
-    def check_buffer_times(self, reservation_unit, begin, end):
-        reservation_after = reservation_unit.get_next_reservation(end, self.instance)
+    def check_buffer_times(
+        self,
+        reservation_unit,
+        begin,
+        end,
+        reservation_type: Optional[ReservationType] = None,
+    ):
+        if (

```

Comment:
This could be also like:

`current_type = getattr(self.instance, "type", reservation_type)`
`if current_type == ReservationType.BLOCKED:`

Reply:
Fixed! 👍